uiowa/uiowa
# [Bug] List items in WYSIWYG preview

Resolves https://github.com/uiowa/uiowa/issues/7803

# How to test

```
ddev blt frontend && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli /node/566/layout
```
1. Edit second block on the page (https://sandbox.prod.drupal.uiowa.edu/node/566/layout) on prod and observe how the lists move outside of the block. 
2. Compare against local site and confirm that lists do not change position
3. Compare /kitchen-sink-0 against prod and confirm no visual regressions
4. Compare /kitchen-sink against prod and confirm no visual regressions
